app/src/main/java/com/github/sdp_begreen/begreen/activities/DatabaseActivity.kt

```diff
@@ -47,7 +47,7 @@ class DatabaseActivity : AppCompatActivity() {
 
             lifecycleScope.launch {
                 imageId = FirebaseDB.addImage(bitmap, 3,
-                    PhotoMetadata(null, null, null, "Plastic bottle"))
+                    PhotoMetadata(null, null, null, null,"Plastic bottle",null))
             }
         }
 
```

Comment:
When I run the tests, this class fails with the following error :

com.github.sdp_begreen.begreen.activities.DatabaseActivityTest > getWithInexistantKeyPrintsNothing[Pixel_2_API_32(AVD) - 12] FAILED

Reply:
As this is not one of my tests we should speak with Luca but locally while running the firebase emulator with the correct import statement I don't have any errors

Reply:
The last PR made by Luca (#40) correct this error. I had exactly the same in my current PR and I correct it by merging the master into my branches (Luca modified only two files). I hope it helps.

Reply:
It is true that I also had to merge the master due to some errors, but I didn't notice that there were the same. Sorry for the confusion